AdministratorAdministration.yaml

```diff
@@ -447,7 +509,7 @@ paths:
                           type: string
                           description: >
                             'Old release number
-                            from [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-http-c-or-1-0-0-000/layer-protocol=0/http-client-interface-1-0:http-client-interface-pac/http-client-interface-configuration/release-number]'
+                            from [/core-model-1-4:control-construct/logical-termination-point=aa-2-0-0-http-s-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/release-number]'
                         subscription:
                           type: string
                           description: >
```

Comment:
line 518 :: the word "and" shall be updated to "or", because the request-body for /v1/end-subscription shall contain either of the requests

Reply:
Added "in two separate, sequential requests."